src/Form.jsx

```diff
@@ -124,7 +124,8 @@ var Form = React.createClass({
         label = field.label,
         formfield = null,
         hasError = this.state.errorElements.indexOf(name) !== -1,
-        inputClass = hasError ? 'error' : '';
+        labelClass = field.labelClassname ? field.labelClassname : '',
```

Comment:
if we make this `... : null` then the label will not even add `class=""` when there is no explicit label class.